include/Dialects/LinalgExt/Passes.h

```diff
@@ -11,17 +11,24 @@
 #include "mlir/Pass/Pass.h"
 
 namespace mlir {
+class DialectRegistry;
+
 namespace linalg_ext {
 
 /// Creates a pass to drive tiling of LinalgExt operations.
 std::unique_ptr<OperationPass<FuncOp>>
 createLinalgExtTilingPass(ArrayRef<int64_t> tileSizes = {});
 
+std::unique_ptr<OperationPass<FuncOp>>
+createLinalgExtTilingToTileOpPass(int64_t tileSize = 0);
```

Comment:
How is that different from the function above?

Reply:
The function above tiles directly to scf.for while still using the interface.

Reply:
Added a commentto distinguish the two.